.github/workflows/checkAutomatedPR.yaml

```diff
@@ -24,14 +27,15 @@ jobs:
           ref: "${{ env.BRANCH_NAME }}"
       - shell: pwsh
         id: ValidateAutomatedPR
+        env:
+          BODY: ${{ github.event.issue.body }}
```

Comment:
Can we add this to above env: section? at line#14

Reply:
yup moved to the global section